app/controllers/reservations_controller.rb

```diff
@@ -9,12 +12,14 @@ def new
   end
 
   def create
-    reservation = CreateReservation.new(params[:email], params[:seance_id], params[:seats])
-    if reservation.call
+    new_reservation = Reservations::UseCases::Create.new(user: current_user, seance_id: params[:seance_id],
```

Comment:
Did you run rubocop? I think it breaks lines differently, but I can be wrong.

Reply:
I did, I even have lefthook so it runs before every commit. This indentation is final effect of `rubocop -a`